src/reference/data-formats.rst

```diff
@@ -30,6 +30,34 @@ When using `tsv-utils <https://opensource.ebay.com/tsv-utils/>`__
     | tsv-uniq -H -f strain \
     | csvtk fix-quotes --tabs > output.tsv
 
+If you are using custom Python scripts to handle TSV files, we recommend using the
+`csv module <https://docs.python.org/3/library/csv.html>`__ to read and write the files.
+
+.. note::
+
+  Be sure to follow `csv module's recommendation <https://docs.python.org/3/library/csv.html#id4>`__
+  to open files with the ``newline=''``.
```

Comment:
_*nit:*_ either remove the `the` here, or add something like `argument` after `newline=''`; otherwise it reads oddly.
  
```suggestion
  to open files with ``newline=''``.
```

OR
  
```suggestion
  to open files with the ``newline=''`` argument.
```